tests/testExamples.m

```diff
@@ -139,7 +139,9 @@ function testWriteScript(testCase, exampleScriptFile)
             close all;
 
             % Write the script
-            problem.writeScript(objName="scriptProblem", script="projectScript");
+            clonedProblem = problem.clone();
+            testCase.verifyNotEqual(clonedProblem, problem);
```

Comment:
Given that this test does not look intuitively correct, I think it might be worth adding a comment stating that this line tests that clone produces a deep copy of the project.